docs/examples/client.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Node packages
+ */
+const didCommon = require('@decentralized-identity/did-common-typescript');
+const didAuth = require('@decentralized-identity/did-auth-jose');
+
+/**
+ * Constants
+ */
+const discoveryEndpoint = 'HTTP_RESOLVER_ENDPOINT_HERE'; // e.g. https://beta.discover.did.microsoft.com/
+const redirectUrl = 'exampleServerUrl.com'; // server url that client will send Authentication Response to
```

Comment:
use `example.com` as the example url. exampleServerUrl may be registered and used as an attack vector in this code sample.
```suggestion
const redirectUrl = 'server.example.com'; // server url that client will send Authentication Response to
```

Reply:
Perhaps better yet, make it usable and send it back to localhost?
```suggestion
const redirectUrl = 'localhost:3000'; // server url that client will send Authentication Response to
```